packages/cli/src/commands/generate/auth/templates/auth.js.template

```diff
@@ -12,10 +75,19 @@ export const getCurrentUser = async (decoded, { token, type }) => {
 }
 
 // Use this function in your services to check that a user is logged in, and
-// optionally raise an error if they're not.
+// raise an error if they're not.
 
 export const requireAuth = () => {
   if (!context.currentUser) {
     throw new AuthenticationError("You don't have permission to do that.")
   }
 }
+
+// Use this function in your services to check whether a user is assigned a role, and
+// raise an error if they have not.
+
+export const hasRole = (role) => {
+  if (!context.currentUser.roles?.includes(role)) {
+    throw new AuthenticationError("You don't have permission to do that.")
```

Comment:
Same here, I think new AuthorizationError would be nice.

Reply:
Done!